sample-linux.ini

```diff
@@ -5,10 +5,12 @@ portNumber = 8000
 ; If any symbols of interest aren't available locally (e.g. Windows DLLs), uncomment line below
 ; remoteSymbolServer = http://symbolapi.mozilla.org:80/
 
-; These options are unlikely to need editing by users
-maxCacheEntries = 2000000
-mruSymbolStateFile = /tmp/snappy-mru-symbols.json
-maxMRUSymbolsPersist = 10
+[MemoryCache]
+maxMemCacheFiles = 400
+
+[DiskCache]
+diskCachePath = /tmp/snappy/cache
+maxDiskCacheFiles = 1500
```

Comment:
Going with more realistic statistics, I created a [small script](https://gist.github.com/walac/fa72f6b15327e7b332a9) to calculate the average file size from production logs. The results showed less than 800KB per file. Being a bit more conservative we can assume 1MB per file, so 1500 files is still a reasonable estimate.